brianlong/www.validators.app
# 186675630 - Prevent from saving DCs without ASN number

#### What's this PR do?
- As in the title.

#### How should this be manually tested?
- `rails test`

Run the following scripts (you should have maxmind access before continue):

- - script/data_centers_scripts/append_data_centers_geo_data.rb
- - script/data_centers_scripts/change_validator_data_center.rb
- - script/data_centers_scripts/check_unknown_data_centers_for_updates.rb
- - script/data_centers_scripts/fix_data_centers_hetzner.rb
- - script/data_centers_scripts/fix_data_centers_ovh.rb
- - script/data_centers_scripts/fix_data_centers_reliablesite.rb
- - script/data_centers_scripts/fix_data_centers_webnx.rb

and verify if any errors have been raised.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186675630)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
